aws/table_aws_cost_by_account_monthly.go

```diff
@@ -19,11 +19,24 @@ func tableAwsCostByLinkedAccountMonthly(_ context.Context) *plugin.Table {
 		Description: "AWS Cost Explorer - Cost by Linked Account (Monthly)",
 		List: &plugin.ListConfig{
 			Hydrate: listCostByLinkedAccountMonthly,
-			Tags:    map[string]string{"service": "ce", "action": "GetCostAndUsage"},
+			KeyColumns: plugin.KeyColumnSlice{
+				{
+					Name:       "period_start",
+					Require:    plugin.Optional,
+					Operators:  []string{">", ">=", "=", "<", "<="},
+					CacheMatch: "exact",
```

Comment:
```suggestion
					CacheMatch: query_cache.CacheMatchExact
```

Reply:
@ParthaI could you please make the change across all the tables?

Reply:
Updated!